chapter11/dtj/IslanderDates.pm

```diff
@@ -0,0 +1,42 @@
+package IslanderDates;
+
+use strict;
+use warnings;
+use Scalar::Util qw ( looks_like_number );
+
+my @day = qw(ark dip wap sen pop sep kir);
+my @month = qw(diz pod bod rod sip wax lin sen kun fiz nap dep);
+sub number_to_day_name {
+    my $number = shift @_;
+    eval {
```

Comment:
~~I wanted the errors to be specific~~

I was debugging a problem and needed the errors to be specific, as I couldn't tell if it was the `number_to_day_name` sub or the `number_to_month_name` sub that was failing (I was calling them both from the same line).